src/olympia/devhub/views.py

```diff
@@ -1467,6 +1467,12 @@ def _submit_upload(
     form = forms.NewUploadForm(
         request.POST or None, request.FILES or None, addon=addon, request=request
     )
+    if wizard or (addon and addon.type == amo.ADDON_STATICTHEME):
+        # If using the wizard or submitting a new version of a theme, we can
+        # force theme_specific to be True. If somehow the developer is not
+        # uploading a theme, validation will reject it just like if they had
+        # tried to use the theme submission flow for an entirely new add-on.
+        theme_specific = True
```

Comment:
submissions via the wizard already set `theme_specific` kwarg, right?  The patch is just covering the case for existing themes having new versions submitted using the xpi upload flow?

Reply:
Yes. I figured it didn't hurt to make sure `theme_specific` is set even if using the wizard.